src/gccrs.md

```diff
@@ -0,0 +1,17 @@
+# `gccrs`
+
+[`gccrs`](https://github.com/rust-gcc/gccrs) is an alternative compiler for the Rust
+programming language, developed from the ground up for the
+[GNU Compiler Collection(GCC)](https://gcc.gnu.org/) project.
+
+While `gccrs` is not yet able to compile Rust code, it is progressing fast - we are hoping we will be able to compile the Rust 1.49 standard library by the next GCC release, 14.1.
+
+Once that milestone is achieved, we will begin catching up to the Rust version used by Rust-for-Linux - in the hopes of being useful quickly!
+
+You can follow the project's progress on [our blogpost](https://rust-gcc.github.io)
```

Comment:
```suggestion
You can follow the project's progress on [our blogpost](https://rust-gcc.github.io).
```

Reply:
This one had a slightly hard to see period suggestion at the end, not sure if you noticed.